thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/v2/DataResource.java

```diff
@@ -543,6 +547,81 @@ public WowSummary getWowSummary(
     return wowSummary;
   }
 
+  @GET
+  @Path("anomalies/ranges")
+  public Map<Long, List<TimeRange>> getAnomalyTimeRangesByMetricIds(
+      @QueryParam("metricIds") String metricIds,
+      @QueryParam("start") Long start,
+      @QueryParam("end") Long end) {
+
+    if (metricIds == null)
+      throw new IllegalArgumentException("Must provide metricIds");
+
+    if (start == null)
+      throw new IllegalArgumentException("Must provide start timestamp");
+
+    if (end == null)
```

Comment:
Stylistic question. Absolutely doable, yet start/stop aren't really properties of the anomaly ranges either (i.e. they're a cutoffs, not anomaly ranges)